docs/reference/generated/slider-root.json

```diff
@@ -2,30 +2,29 @@
   "name": "SliderRoot",
   "description": "Groups all parts of the slider.\nRenders a `<div>` element.",
   "props": {
-    "aria-labelledby": {
```

Comment:
This prop is missing.

Reply:
The API extractor documents only properties that are defined in the project it analyses (otherwise we'd have a ton of HTML attributes in all components).